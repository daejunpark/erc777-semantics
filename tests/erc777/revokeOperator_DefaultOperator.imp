test.preamble();

// Testing revokeOperator(Operator) when Operator is an default operator
// should pass

test.revokeOperator(8);
