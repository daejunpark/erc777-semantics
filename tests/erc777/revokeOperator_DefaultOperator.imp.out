7 is revoking operator 8
